app/src/main/java/com/example/pygmyhippo/user/ViewMyEventFragment.java

```diff
@@ -0,0 +1,287 @@
+package com.example.pygmyhippo.user;
```

Comment:
My Initial curiosity is if we let users navigate to this page instead of the old entrant fragment, then how do they have the ability to leave the event if they aren't invited?

Reply:
NVM I'm just BLIND. I'll have to refactor my geolocation for this file instead of EventFragment so we should probably merge this first